src/context/videoContext.jsx

```diff
@@ -0,0 +1,40 @@
+import { createContext, useContext, useEffect, useReducer, useState } from 'react';
+import VideoReducer from "reducer/videoReducer";
+import { getVideos } from "services/videoService"
+
+const VideoContext = createContext();
+
+const VideoProvider = ({ children }) => {
+    const [loader, setLoader] = useState(false)
```

Comment:
Instead set loader initially to true and only make it false after videoDispatch()

Reply:
Thanks for suggestion